jenkins/databricks/create.py

```diff
@@ -34,14 +34,17 @@ def main():
   worker_type = 'g4dn.xlarge'
   driver_type = 'g4dn.xlarge'
   cloud_provider = 'aws'
+  # comma seperated init scritps, e.g., dbfs:/foo,dbfs:/bar,...
+  init_script = ''
```

Comment:
nit: rename to be init_scripts (ie with an s) everywhere

Reply:
fixed